### DISCUSSION TITLE
Welcome to automatoes Discussions!

### DISCUSSION THREAD

**Discussion by piraz** [candango/automatoes#80] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by szepeviktor** [comment]
@piraz How about moving gitter chat here?

**Reply by piraz** [reply]
I agree.

**Reply by piraz** [reply]
Done: 96de0d5252a69c5d166f1e34bb29f75bd5c07e9a
